src/tinybox.vue

```diff
@@ -202,6 +202,30 @@ export default {
         window.removeEventListener('keyup', this.keyup);
       }
     },
+    
+    /*
+     * Center the thumbnails' scrollbar to the clicked image
+     */
+    index: function(index) {
+      if (!this.noThumbs) {
+        Vue.nextTick(function() {
+          if (typeof(index) != 'undefined' && index != null) {
```

Comment:
Codacy found an issue: [Unary word operator 'typeof' must be followed by whitespace.](https://app.codacy.com/manual/NickKaramoff/vue-tinybox/pullRequest?prid=5742748)